<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table_matrix</class>
 <widget class="QWidget" name="Table_matrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Матрица исходных данных</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>11</y>
     <width>651</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить таблицу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
